next/src/components/sections/InbaLatestReleaseSection.tsx

```diff
@@ -0,0 +1,152 @@
+import { Typography } from '@bratislava/component-library'
+import { keepPreviousData, useQuery } from '@tanstack/react-query'
+import { useTranslation } from 'next-i18next'
+import React, { Fragment } from 'react'
+
+import ArticleRowCard from '@/src/components/cards/ArticleRowCard'
+import Button from '@/src/components/common/Button/Button'
+import HorizontalDivider from '@/src/components/common/Divider/HorizontalDivider'
+import CardImage from '@/src/components/common/Image/CardImage'
+import LoadingSpinner from '@/src/components/common/LoadingSpinner/LoadingSpinner'
+import SectionContainer from '@/src/components/layouts/SectionContainer'
+import SectionHeader from '@/src/components/layouts/SectionHeader'
+import { InbaLatestReleaseSectionFragment } from '@/src/services/graphql'
+import { client } from '@/src/services/graphql/gql'
+import { formatDate } from '@/src/utils/formatDate'
+import { generateImageSizes } from '@/src/utils/generateImageSizes'
+import { getLinkProps } from '@/src/utils/getLinkProps'
+import { isDefined } from '@/src/utils/isDefined'
+
+export const latestInbaReleaseQueryOptions = {
+  queryKey: ['Latest Inba Release'],
```

Comment:
Usually we do not use spaces, but the actual query name, which would be `LatestInbaRelease` (from `client.LatestInbaRelease()`)